sideboard/lib/_cp.py

```diff
@@ -60,7 +98,7 @@ def mainloop():
     try:
         while not stopped.is_set():
             try:
-                stopped.wait(0.1)
+                stopped.wait(config['thread_wait_interval'])
```

Comment:
We made this configurable and also changed the default value from 0.1 seconds to 1 second, for performance reasons.  We found that when running hundreds of background threads then 0.1 seconds was little enough to noticably affect the CPU, while 1.0 seconds didn't slow anything down in a meaningful way.